# Object Coordinates
object_coordinates_x: [-0.16, -0.16, 0.32]
object_coordinates_y: [-0.27713, 0.27713, 0.0]

# Object Scale (Need to be the same as state in the object urdf file)
scale_x: 0.98
scale_y: 0.98
# scale_x: 1.259320
# scale_y: 1.007450
# scale_x: 0.8
# scale_y: 0.8
# scale_x: 2.4
# scale_y: 2.4
# scale_x: 2.51864
# scale_y: 2.0149